### PR TITLE
26.0.0 beta 3, 25.0.4 rc1, 24.0.10 rc1

### PR DESCRIPTION
**Pull request by blizzz**

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -603,101 +603,101 @@
 	'beta' => [
 		'26' => [
 			'100' => [
-				'latest' => '26.0.0 beta 2',
-				'internalVersion' => '26.0.0.4',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta2.zip',
+				'latest' => '26.0.0 beta 3',
+				'internalVersion' => '26.0.0.5',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta3.tar.bz2',
```
Comment (AndyScherzinger) [inline config/config.php:608]: shouldn't it be zip?

File: config/config.php [hunk 2]
```diff
@@ -603,101 +603,101 @@
...
 		],
-		'25.0.3' => [
+		'25.0.4' => [
 			'100' => [
-				'latest' => '26.0.0 beta 2',
-				'internalVersion' => '26.0.0.4',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta2.zip',
+				'latest' => '26.0.0 beta 3',
+				'internalVersion' => '26.0.0.5',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta3.tar.bz2',
```
Comment (AndyScherzinger) [inline config/config.php:624]: same here: zip?

File: tests/integration/features/beta.feature [hunk 3]
```diff
@@ -670,16 +670,16 @@ Feature: Testing the update scenario of beta releases
     And the installation mtime is "11"
     When The request is sent
     Then The response is non-empty
-    And Update to version "26.0.0.4" is available
-    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta2.zip"
+    And Update to version "26.0.0.5" is available
+    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-26.0.0beta3.tar.bz2"
```
Comment (AndyScherzinger) [inline tests/integration/features/beta.feature:674]: zip?

File: config/config.php [hunk 4]
```diff
@@ -603,101 +603,101 @@
...
 
-		'24.0.9' => [
+		'24.0.10' => [
 			'100' => [
-				'latest' => '25.0.3',
-				'internalVersion' => '25.0.3.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-25.0.3.zip',
+				'latest' => '25.0.4 RC1',
+				'internalVersion' => '25.0.4.0',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-25-stable.zip',
```
Comment (nickvergessen) [inline config/config.php:657]: broken dl url?

Comment (nickvergessen) [reply config/config.php:657]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-25-4rc1.zip',
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: config/config.php (hunk 4 above)
Comment (blizzz) [reply config/config.php:657]: 😳 i swear i fixed it before pushing

Comment (blizzz) [reply config/config.php]: > flushed i swear i fixed it before pushing

… just not everywhere 😳

**Comment by blizzz** [comment]
tests as expected

**Pull request merged**